frontend/src/Commons/Forms/InputModules/FieldsAndSoil/FieldsListComponent.tsx

```diff
@@ -52,73 +53,77 @@ const FieldsListComponent: React.FC<FieldListProps> = ({ farmDetails }) => {
+                    </p>
+                  </div>
+                </StyledListItem>
+                <StyledListItem width="180px">
+                  <div className="smallItems">
+                    <h2>P (ppm)</h2>
+                    <p key={field.SoilTest.ValP}>
+                      {field.HasSoilTest ? field.SoilTest.ValP : highValue}
+                    </p>
+                  </div>
+                </StyledListItem>
+              </StyledListItemGroup>
+              <StyledListItemGroup>
+                <StyledListItem width="180px">
+                  <div className="smallItems">
+                    <h2>K (ppm)</h2>
+                    <p key={field.SoilTest.valK}>
+                      {field.HasSoilTest ? field.SoilTest.valK : highValue}
+                    </p>
+                  </div>
+                </StyledListItem>
+                <StyledListItem width="180px">
+                  <div className="smallItems">
+                    <h2>pH</h2>
+                    <p key={field.SoilTest.valPH}>
+                      {field.HasSoilTest ? field.SoilTest.valPH : highPH}
+                    </p>
+                  </div>
+                </StyledListItem>
+              </StyledListItemGroup>
+              <StyledListItemGroup>
+                <StyledListItem width="180px">
+                  <h2>Leaf Tissue P(%)</h2>
+                  <p key={field.LeafTest.leafTissueP}>
+                    {field.HasLeafTest ? field.LeafTest.leafTissueP : highValue}
                   </p>
-                </div>
-              </StyledListItem>
-              <StyledListItem width="180px">
-                <div className="smallItems">
-                  <h2>pH</h2>
-                  <p key={fields.SoilTest.valPH}>
-                    {fields.hasSoilTest ? fields.SoilTest.valPH : highPH}
+                </StyledListItem>
+                <StyledListItem width="180px">
+                  <h2>Leaf Tissue K(%)</h2>
+                  <p key={field.LeafTest.leafTissueK}>
+                    {field.HasLeafTest ? field.LeafTest.leafTissueK : highValue}
                   </p>
-                </div>
-              </StyledListItem>
-            </StyledListItemGroup>
-          </StyledListItemGroupContainer>
-          <StyledListItemGroup>
-            <StyledListItem width="180px">
-              <h2>Leaf Tissue P(%)</h2>
-              <p key={fields.LeafTest.leafTissueP}>
-                {fields.hasLeafTest ? fields.LeafTest.leafTissueP : highValue}
-              </p>
-            </StyledListItem>
-            <StyledListItem width="180px">
-              <h2>Leaf Tissue K(%)</h2>
-              <p key={fields.LeafTest.leafTissueK}>
-                {fields.hasLeafTest ? fields.LeafTest.leafTissueK : highValue}
-              </p>
-            </StyledListItem>
-          </StyledListItemGroup>
+                </StyledListItem>
+              </StyledListItemGroup>
```

Comment:
This should be after Line 124 to achieve the Leaf Test list to align with the mockups.

Reply:
I removed it while debugging and readded to the wrong place